Improve docs on how to install logcli from source (#5126)
grafana/loki: docs/sources/getting-started/logcli.md

Download the `logcli` binary from the
[Loki releases page](https://github.com/grafana/loki/releases).

> ### Compile from source

Reviewer: ```suggestion
### Build logcli from source
```

> Clone Github repository and compile `logcli` from source:

Reviewer: ```suggestion
Clone the Loki repository and build `logcli` from source:
```

> ```bash
> git clone https://github.com/grafana/loki.git
> cd loki
> make logcli
> # optionally, move the binary into a directory that is part of $PATH
> cp cmd/logcli/logcli /usr/local/bin/logcli

Reviewer: ```suggestion
```bash
git clone https://github.com/grafana/loki.git
cd loki
make logcli
```

Optionally, move the binary into a directory that is part of your `$PATH`:

```bash
cp cmd/logcli/logcli /usr/local/bin/logcli
```
